### PR TITLE
_colcon_ordered_commands might be null

### PR DESCRIPTION
**Pull request by ivanpauno**
I really haven't read what `_local_setup_util_ps1.py` is doing, but when sourcing `local_setup.ps1` in some workspaces I get:

> You cannot call a method on a null-valued expression.
> At C:\Users\Administrator\Documents\testing_ivanpauno\tutorials_ws\install\local_setup.ps1:53 char:1
> + $_colcon_ordered_commands.Split([Environment]::NewLine, [StringSplitO ...
> + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
>    + FullyQualifiedErrorId : InvokeMethodOnNull

This PR solves the issue.

---

My guess is that `_local_setup_util_ps1.py` is avoiding to source files that were already source before (or sth like that), and that sometimes end up with a `null` result (?).

### CODE REVIEW

**Comment by ivanpauno** [comment]
@jacobperron I think you're a maintainer of this.

I ran into the issue when following this tutorial https://docs.ros.org/en/rolling/Tutorials/Writing-A-Simple-Py-Publisher-And-Subscriber.html.

**Review by jacobperron — APPROVED** [review]
Thanks for the fix!

**Comment by jacobperron** [comment]
Looks like the appveyor flake8 errors are unrelated.

**Comment by dirk-thomas** [comment]
> Looks like the appveyor flake8 errors are unrelated.

Please do not merge any PRs with failing CI. In the future address unrelated CI failures in a separate PR and then rebase the one which failed CI before.

**Pull request merged**
